spec/features/support/providers/locations/creating_multiple_locations_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+feature 'Multiple locations' do
+  before do
+    given_i_am_authenticated(user: create(:user, :admin))
+  end
+
+  scenario 'submitting an empty form' do
+    given_the_multiple_locations_feature_flag_is_active
```

Comment:
this should probably read `and_*` since the step before is `given` otherwise it reads a bit awkwardly

Reply:
updated: https://github.com/DFE-Digital/publish-teacher-training/pull/3398/commits/3f0d9542600c44cc5ec950407295dac98bb73078